nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -283,7 +283,16 @@ int ddxProcessArgument(int argc, char *argv[], int i)
 
     if (nxagentOptionFile)
     {
-      nxagentProcessOptionsFile();
+      /* if the "filename" starts with nx/nx treat it
+	 as an option string */
+      if (strncasecmp(nxagentOptionFile, "nx/nx", 5) == 0)
+      {
```

Comment:
I was thinking about checking for all 4 of these. But I consider most of them as obsolete, nowadays I only see "nx/nx". Also I am not really happy with using nx/nx here at all, it feels like misuse here. Maybe we should simply interpret a `:` at the beginning as marker for a string.